core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -1097,7 +1100,9 @@ private[spark] object JsonProtocolSuite extends Assertions {
       |  },
       |  "Task Metrics": {
       |    "Executor Deserialize Time": 300,
+      |    "Executor Deserialize CPU Time": 0,
```

Comment:
Yeah I tested it on my testing cluster, but this makes sense. I will add non-zero CPU times by setting the CPU times same as given wall times.